README.md

```diff
@@ -1,75 +1,44 @@
-[<img align="right" src="https://img.shields.io/github/issues/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/issues)
-[<img align="right" src="https://img.shields.io/github/license/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/blob/master/LICENSE)
-[<img align="right" src="https://img.shields.io/github/forks/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/network/members)
-[<img align="right" src="https://img.shields.io/github/stars/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/stargazers)
-[<img align="right" src="https://img.shields.io/github/release/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/releases)
+[ ![Get Addon][amo-button] ][amo-website]
```

Comment:
I suppose this is probably a personal preference, but I liked it when the button what anchored on the right instead of the left.

Reply:
I prefer at right too... Also, we will remove all the badges too? Maybe we can keep 1 or 2?

Reply:
I think the only relevant badge would be the version number badge. All of the others are already on the Github UI.

Reply:
yes, makes sense.. 🙂

Reply:
> I liked it when the button what anchored on the right instead of the left.

Same. However, I wanted to remove the HTML and Markdown doesn't support text alignment. 

> I think the only relevant badge would be the version number badge. All of the others are already on the Github UI.

I've readded the version badge.